### ISSUE TITLE
Documentation 

### ISSUE THREAD

**Issue by thomas-ecosurv** [rebing/graphql-laravel#726]
For example, the use of the model attribute is not documented.  

    protected $attributes = [
        'name'          => 'Profile',
        'description'   => 'A user profile',
        'model'         => UserProfileModel::class,
    ];

If you can provide some details Ill do a PR on the README.md with updated docs to help :)

**Comment by thomas-ecosurv** [comment]
For example, the use of the model attribute is not documented.  

    protected $attributes = [
        'name'          => 'Profile',
        'description'   => 'A user profile',
        'model'         => UserProfileModel::class,
    ];

If you can provide some details Ill do a PR on the README.md with updated docs to help :)

**Comment by mfn** [comment]
I take all the PRs I can get!

(ok, they are unmerged ones… I mean, docs are less work/controversial to review 😄 )

I'm closing this issue, not because of disregarding your request (see above sentence 🖕) but because I don't believe for the amount of contributors currently active there's not much value in it.

TL;DR: just make PRs `\o/`
